docker/Dockerfile.ci_cpu

```diff
@@ -10,13 +10,6 @@ RUN bash /install/ubuntu_install_core.sh
 COPY install/ubuntu_install_build.sh /install/ubuntu_install_build.sh
 RUN bash /install/ubuntu_install_build.sh
 
-# ANTLR deps
-COPY install/ubuntu_install_java.sh /install/ubuntu_install_java.sh
-RUN bash /install/ubuntu_install_java.sh
-
-COPY install/ubuntu_install_antlr.sh /install/ubuntu_install_antlr.sh
-RUN bash /install/ubuntu_install_antlr.sh
-
```

Comment:
Why delete? I think this is required by Jenkins?

Reply:
Why jenkins need this? ANTLR is a AST analysis tool? 
I'll remove all ci related change in this PR and put it at https://github.com/dmlc/dgl/pull/1054